sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -832,6 +832,14 @@ object SQLConf {
       .intConf
       .createWithDefault(10000)
 
+  val IGNORE_DATA_LOCALITY =
+    buildConf("spark.sql.sources.ignore.datalocality")
+      .doc("If it is set to true, Spark will not fetch the block locations for each file on " +
+        "listing files, which will greatly accelerate the list file operation, while loss " +
```

Comment:
`while loss data locality`? Could you revise the syntax?

Reply:
Resolved.